e2e/testdata/hydration/helm-components/kustomization.yaml

```diff
@@ -22,7 +22,7 @@ helmCharts:
   releaseName: my-wordpress
   namespace: wordpress
   valuesInline:
-    existingSecret: my-wordpress
+    wordpressPassword: abcdefg
```

Comment:
FYI this was changed because the helm chart never reconciled before, the previous configuration assumed the Secret was created out of band.

This makes it so that the helm chart reconciles and the ResourceGroup passes its validation

Reply:
maybe pull this out to its own PR with its own description, along with the new files you added which i assume are for the same reason.

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1595